src/igvfd/upgrade/treatment.py

```diff
@@ -10,3 +10,9 @@ def treatment_1_2(value, system):
     if 'documents' in value:
         if len(value['documents']) == 0:
             del value['documents']
+
+
+@upgrade_step('treatment', '2', '3')
```

Comment:
i think cleaner would be to turn any treatment not specifying purpose to be "perturbation" and add that info to "notes".